payments/views.py

```diff
@@ -112,34 +103,37 @@ def post(self, request, *args, **kwargs):
             # Invalid signature
             return HttpResponse(status=400)
 
-        #get payload and type of object that came in the event
+        # get payload and type of object that came in the event
         event_body, object_type = get_event_payload_and_type(event)
-        #only checkout_session, payment_intent and charge objects come back with metadata
+        # only checkout_session, payment_intent and charge objects come back with metadata
         if getattr(event_body.metadata, 'topup_pk', None):
-            topup = get_transaction_record(event_body) # find transaction's record in database
-            save_id_and_status(event_body, topup, object_type) # save event's id and status
+            topup = get_transaction_record(event_body)  # find transaction's record in database
+            save_id_and_status(event_body, topup, object_type)  # save event's id and status
             if event.type == 'payment_intent.created':
-                save_amount_data(event_body, topup) 
-                is_live_mode(event_body, topup) # flags if test or live
+                save_amount_data(event_body, topup)
+                is_live_mode(event_body, topup)  # flags if test or live
             elif event.type == 'payment_intent.succeeded':
-                increase_balance(event_body, topup) # add funds to user's account
+                increase_balance(event_body, topup)  # add funds to user's account
             topup.save()
         return HttpResponse(status=200)
 
+
 def get_event_payload_and_type(event):
     event_body = event.data.object
     object_type = event_body.object
     return event_body, object_type
 
+
 def get_transaction_record(event_body):
     topup_pk = event_body.metadata.topup_pk
     try:
-        topup = pay_models.TopUp.payments.get(pk=topup_pk)
+        topup = TopUp.payments.get(pk=topup_pk)
         return topup
-    except pay_models.TopUp.DoesNotExist:
-        print('record with this pk does not exist') #TODO handle error
+    except TopUp.DoesNotExist:
+        print('record with this pk does not exist')  # TODO handle error
```

Comment:
Print function is not recommended it this type application. Please use logger if is needed :D